probemaker.py

```diff
@@ -133,7 +134,35 @@ def __init__(self, file, show_values=False):
             print(self.tal)
             print('=======================')
 
-    def probe(self, talent: str, mod: int=0):
+    def perform_attr_probe(self, attr: str, mod: int = 0):
+        print(f"The mighty {self.name} has incredible {self.attr[attr]} points in {attr}," +
+              f"the modifier for this probe is {mod}")
+        roll = randint(1, 20)
+        res = self.attr[attr] - roll + mod
+        print(f'The die shows a {roll}')
+        if res >= 0 and roll != 20:
+            print(f"{self.name} has passed")
+            if roll == 1:
+                print('Will it be meisterlich?')
+                roll2 = randint(1, 20)
+                res2 = self.attr[attr] - roll2 + mod
+                if res >= 0:
+                    print('Yes!')
+                else:
+                    print('No :(')
+        elif roll != 20:
+            print(f"{self.name} has failed")
+
+        if roll == 20:
+            print(f"{self.name} has failed, will it be a complete disaster?")
+            roll2 = randint(1, 20)
+            res2 = self.attr[attr] - roll2 + mod
+            if res <= 0:
+                print("Yes....")
+            else:
```

Comment:
maybe this method should return the actual result value (useful for logging/storing later)

Reply:
Jepp, lets do this though once we start logging.